source/extensions/filters/listener/proxy_protocol/proxy_protocol_header.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+namespace Envoy {
+namespace Extensions {
+namespace ListenerFilters {
+namespace ProxyProtocol {
+
+//
+// See https://github.com/haproxy/haproxy/blob/master/doc/proxy-protocol.txt
+// for definitions
+
+constexpr char PP1_SIGNATURE[] = "PROXY ";
```

Comment:
nit: generally prefer longer and more readable named. E.g., PROXY_PROTO_V1_SIGNATURE, etc.